src/bindings/jna/gradlew

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Please also add this to doc/THIRD-PARTY-LICENSES

Reply:
will do

Reply:
@markus2330 done. do i need to list newly used third party software somewhere? (gradle, a few gradle plugins)